p2p/node.go

```diff
@@ -231,6 +232,13 @@ func (n *QriNode) GoOffline() error {
 	return err
 }
 
+// ResolveRefLocally attempts to resolve a dataset using the local repo
+func (n *QriNode) ResolveRefLocally(ctx context.Context, ref *dsref.Ref) error {
+	localResolver := dsref.SequentialResolver(n.Repo.Dscache(), n.Repo)
+	_, err := localResolver.ResolveRef(ctx, ref)
+	return err
+}
+
```

Comment:
I don't think we should keep this method

Reply:
ahhh thank you for catching this